src/ui/server/controllers/insurance/feedback/feedback-form/index.ts

```diff
@@ -95,9 +111,20 @@ const post = async (req: Request, res: Response) => {
         [OTHER_COMMENTS]: feedback[OTHER_COMMENTS],
       } as InsuranceFeedbackVariables;
 
+      // variables for adding feedback to the database
+      const databaseVariables = {
```

Comment:
`feedbackVariables` please, we shouldn't be mentioned DB in the UI, the UI only interacts with the API